packages/@lwc/compiler/src/compiler/compiler.ts

```diff
@@ -9,9 +9,10 @@ import { CompilerDiagnostic, DiagnosticLevel } from '@lwc/errors';
 import { bundle } from '../bundler/bundler';
 import {
     CompilerOptions,
-    validateOptions,
+    validateCompilerOptions,
     normalizeOptions,
```

Comment:
we should rename this to normalizeCompilerOptions to stay consistent with the naming

Reply:
But then it would imply that there should be a corresponding `validateTransformOptions` whereas this is good for both.

Reply:
that should be fine, validateCompilerOption would validated both transform portion of the options and files, which belong to the compiler; whereas validateTransformOptions would only be responsible for transform options and will be invoked directly from the transform flow.